selfdrive/car/tests/big_cars_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+MAX_EXAMPLES=300
+INTERNAL_SEG_CNT=300
+FILEREADER_CACHE=1
+INTERNAL_SEG_LIST=selfdrive/car/tests/test_models_segs.txt
+
+cd selfdrive/car/tests && pytest test_models.py test_car_interfaces.py
```

Comment:
`./big_cars_test.sh: line 8: cd: selfdrive/car/tests: No such file or directory`

Reply:
ah, I ran these from the base directory, just need a cd to basedir at the front